src/libraries/System.Net.Http/tests/StressTests/HttpStress/run-docker-compose.ps1

```diff
@@ -57,7 +57,7 @@ if ($useWindowsContainers)
     $env:DOCKERFILE="windows.Dockerfile"
 }
 
-docker-compose --file "$COMPOSE_FILE" build $BUILD_ARGS.Split()
+(docker-compose --file "$COMPOSE_FILE" build $BUILD_ARGS.Split()) 2> $null
```

Comment:
Below code should solve problem with PowerShell interpreting stderr output as error and it will also handle any actual errors coming from `docker-compose`:

```suggestion
$originalErrorPreference = $ErrorActionPreference
$ErrorActionPreference = 'Continue'
try {
	docker-compose --file "$COMPOSE_FILE" build $BUILD_ARGS.Split() 2>&1 | ForEach-Object { "$_" }
	if ($LASTEXITCODE -ne 0) {
		throw "docker-compose exited with error code $LASTEXITCODE"
	}
}
finally {
	$ErrorActionPreference = $originalErrorPreference
}
```